config/environments/production.rb

```diff
@@ -83,16 +83,17 @@
   config.active_record.dump_schema_after_migration = false
 
   config.action_mailer.delivery_method = :smtp
+  config.action_mailer.default_url_options = { host: "http://paired-be.herokuapp.com" }
   config.action_mailer.perform_deliveries = true
   config.action_mailer.smtp_settings = {
-   :user_name => ENV['SENDGRID_USERNAME'],
-   :password => ENV['SENDGRID_PASSWORD'],
-   :domain => 'yourdomain.com',
+   :user_name => 'apikey',
```

Comment:
Thanks for digging in and figuring out the problem! I'm glad it was a relatively easy fixed once the problem was located.

Reply:
well...here's hoping it's fixed. :D